cogs/weather.py

```diff
@@ -0,0 +1,51 @@
+import discord
```

Comment:
Don't forget to import the `os` library, if we want to use environment variables: 
```suggestion
import os
import discord
```